dev-tools/commands/src/upload-track.mjs

```diff
@@ -101,7 +101,7 @@ program.command("upload-track")
             ? JSON.parse(premiumConditions)
             : null,
           ai_attribution_user_id: null,
-          preview_start_seconds: previewStartSeconds,
+          preview_start_seconds: parseInt(previewStartSeconds),
```

Comment:
I find myself wondering if the tool we use to parse command arguments is able to parse numbers for us. I made this mistake 3 times in my last PR.

Reply:
https://github.com/tj/commander.js#custom-argument-processing looks like there's this, which we could use. But I kinda like how totally lightweight these commands are